src/status_im/components/styles.cljs

```diff
@@ -26,6 +28,7 @@
 (def color-gray8 "#6E777E")
 (def color-gray9 "#E9EBEC")
 (def color-gray10 "#9BA3A8")
+(def color-gray11 "rgb(238, 241, 245)")
```

Comment:
could you convert the rgb to hex ? we usually don't put rgb here

Reply:
done